src/evaluate/loading.py

```diff
@@ -131,6 +131,15 @@ def convert_github_url(url_path: str) -> Tuple[str, Optional[str]]:
     return url_path, sub_directory
 
 
+def convert_hf_hub_path(path, branch="main"):
+    """Finds raw version of some file in the form {USER/REPO/FILE.FILETYPE}"""
+    repo_owner = path.split("/")[0]
+    repo_name = path.split("/")[1]
+    rest_of_file = "/".join(path.split("/")[2:])
+    url_path = f"https://huggingface.co/datasets/{repo_owner}/{repo_name}/raw/{branch}/{rest_of_file}"
```

Comment:
maybe you can add this to config and use it like in the hub urls here: https://github.com/huggingface/evaluate/blob/c447fc8eda9c62af501bfdc6988919571050d950/src/evaluate/utils/file_utils.py#L99